module-2/lab-mongo/Mongo.ipynb

```diff
@@ -0,0 +1,582 @@
+{
```

Comment:
- I encourage you to complete this lab when you have a free time. 
- I invite you to explore the aggregate () function in mongo, similar to groupby. Play with this data set to get some aggregations from Mongodb.
- Remember to ask Marc to solve question number 12!!